io/example/mets.xml

```diff
@@ -1,126 +1,350 @@
+               ”OCR-D-SEG-DOC”       Document analysis. (Module 2, submodule D: document analysis)
+     -->
+   <mets:fileGrp USE="OCR-D-SEG-DOC">
+      <mets:file ID="FILE_0001_SEG_DOC" GROUPID="PAGE_0001" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000001_SEG_DOC.xml" />
+      </mets:file>
+      <mets:file ID="FILE_0002_SEG_DOC" GROUPID="PAGE_0002" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000002_SEG_DOC.xml" />
+      </mets:file>
+    </mets:fileGrp>
+     <!-- 
+               ”OCR-D-OCR-TESS"      OCR with Tesseract. (Module 3, submodule A: OCR)
+    -->
+    <mets:fileGrp USE="OCR-D-OCR-TESS">
+      <mets:file ID="FILE_0001_OCR_TESS" GROUPID="PAGE_0001" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000001_OCR_TESS.xml" />
+      </mets:file>
+      <mets:file ID="FILE_0002_OCR_TESS" GROUPID="PAGE_0002" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000002_OCR_TESS.xml" />
+      </mets:file>
+    </mets:fileGrp>
+     <!-- 
+               "OCR-D-OCR-ANY"       OCR with any other language. (Module 3, submodule A: OCR)
+    -->
+    <mets:fileGrp USE="OCR-D-OCR-ANY">
+      <mets:file ID="FILE_0001_OCR_ANY" GROUPID="PAGE_0001" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000001_OCR_ANY.xml" />
+      </mets:file>
+      <mets:file ID="FILE_0002_OCR_ANY" GROUPID="PAGE_0002" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000002_OCR_ANY.xml" />
+      </mets:file>
+    </mets:fileGrp>
+     <!-- 
+               "OCR-D-COR-CIS"       CIS post correction. (Module 3, submodule B: OCR)
+    -->
+    <mets:fileGrp USE="OCR-D-COR-CIS">
+      <mets:file ID="FILE_0001_COR_CIS" GROUPID="PAGE_0001" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000001_COR_CIS.xml" />
+      </mets:file>
+      <mets:file ID="FILE_0002_COR_CIS" GROUPID="PAGE_0002" MIMETYPE="text/xml">
+        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000002_COR_CIS.xml" />
+      </mets:file>
+    </mets:fileGrp>
+     <!-- 
+               "OCR-D-COR-ASC"       ASC post correction. (Module 3, submodule B: OCR)
+    -->
+    <mets:fileGrp USE="OCR-D-COR-ASC">
```

Comment:
This should be USE="OCR-D-COR-ASV" (ASV is for Institut für Automatische Sprachverarbeitung)

Reply:
You are right. It's a typing error.